src/pages/documentation/adobe-analytics/migrate-to-edge-network.md

```diff
@@ -0,0 +1,88 @@
+| 5b. **In your app code**, also register the **Edge Bridge extension** with registering *Edge Network* and *Identity for Edge Network extensions*. |  | ✅ |
+| 6. **Remove the Analytics extension** dependency and extension registration from your mobile app code. <br/> **NOTE:** You should still keep the Analytics extension installed in the mobile property (tag) to ensure published versions of your app (pre-migration) continue to work seamlessly. |  ✅ | ✅ |
+| 7 .**Use Edge.sendEvent API** to send data in XDM format to Edge Network based on the schema you have defined. |  ✅ |  |
+| 8. **Keep existing MobileCore.trackAction / MobileCore.trackState API calls** to send data in context data format to Experience Platform.|  | ✅ |
+| 9. **Map your context data to XDM** in Data Prep for Data Collection.|  | ✅ |
+
+
+### Implement the Edge Network extension
+
+To learn about the end to end Edge Network extension implementation process, follow the tutorials below:
+- [iOS tutorial](https://github.com/adobe/aepsdk-edge-ios/tree/main/Documentation/Tutorials)
+- [Android tutorial](https://github.com/adobe/aepsdk-edge-android/tree/main/Documentation/Tutorials)
+
+### Implement the Edge Bridge extension
+
+To learn about the end to end Edge Bridge extension implementation process, follow the tutorials below:
+- [iOS tutorial](https://github.com/adobe/aepsdk-edgebridge-ios/tree/main/Documentation/tutorials)    
+- [Android tutorial](https://github.com/adobe/aepsdk-edgebridge-android/tree/main/Documentation/tutorials) 
+  
+## Other migration considerations
+
+### Not yet on latest Mobile SDK? 
+If you are not using the latest version of the Mobile SDK (that is, still on v4 Mobile SDK or ACP-prefix Mobile SDK), we recommend that you first migrate to the [latest version](https://developer.adobe.com/client-sdks/documentation/current-sdk-versions), and then continue with this guide.
+
+### Consent for Edge Network extension
+The Consent for Edge Network mobile extension enables consent preference collection from your mobile app when using the Experience Platform Mobile SDK and Edge Network.
+
+The privacy status setting (MobilePrivacyStatus.OPT_IN, OPT_OUT, UNKNOWN) set in the mobile tag property which is used by Adobe Experience Cloud mobile extensions (for example, Analytics and Target), are not read nor used by the Edge Network extensions. The Edge Network extensions rely on the Consent for Edge Network extension for this setting.
+
+For more details, please refer to the [Consent for Edge Network extension documentation](https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network).
+
+
+### Identity consideration for the migration
+When using Identity for Edge Network extension, the Mobile SDK automatically migrates any existing MID/ECID to the new XDM IdentityMap format out of the box. If you are using custom identities with `syncIdentifier` / `syncIdentifiers` APIs, you should consider migrating your IDs to the XDM format.
```

Comment:
- any existing MID/ECID -> an existing ECID (previously known as MID) 
- custom identities with **the** ...